layout/MDErgo1-Default-1.kll

```diff
@@ -29,8 +30,9 @@ U"0" : U"F10";
 U"O" : CONS"BRIGHTNESSDECREMENT";
 U"P" : CONS"BRIGHTNESSINCREMENT";
 U"L" : U"UP";
-U"FUNCTION7" : U"FUN2";
```

Comment:
I thought it would be cool to be able to use this ƒ1 as an ƒ2 by hitting the _other ƒ1_ key located in the bottom right. unfortunately this doesn't seem to work 😕